src/tests/ftest/daos_test/daos_core_test_dfs.yaml

```diff
@@ -0,0 +1,59 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+  test_clients:
+    - client-E
+    - client-F
+
+timeout: 600
+pool:
+  #This will create 8G of SCM and 16G of NVMe size of pool.
```

Comment:
Does it create a pool with 16G NVMe?

Reply:
Yes it will. 'daos_test' automatically created nvme of double the size of scm specified.

Reply:
I see. Thanks